### DISCUSSION TITLE
Slipsuit doesnt have new animation.

### DISCUSSION THREAD

**Discussion by Ydrec** [FakeFishGames/Barotrauma#13814] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Slipsuits don't have

TriggerAnimation Type="Walk" FileName="HumanWalkDivingSuit" priority="1" ExpectedSpecies="Human"
TriggerAnimation Type="Run" FileName="HumanRunDivingSuit" priority="1" ExpectedSpecies="Human"

### Reproduction steps

Look at slipsuit xml

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
They aren't needed, because Slipsuit is a variant of the Combat Diving Suit and inherits the animations from it. Giving this a test, the animations seem to be working fine.
